src/polyfill/api_only.js

```diff
@@ -40,31 +40,31 @@ function setupPolyfill() {
 
   const publicApi = Object.create(TrustedTypePolicyFactory.prototype);
   Object.assign(publicApi, {
-    'isHTML': tt.isHTML,
-    'isScriptURL': tt.isScriptURL,
-    'isScript': tt.isScript,
-    'createPolicy': tt.createPolicy,
-    'getAttributeType': tt.getAttributeType,
-    'getPropertyType': tt.getPropertyType,
-    'getTypeMapping': tt.getTypeMapping,
-    'emptyHTML': tt.emptyHTML,
-    'emptyScript': tt.emptyScript,
-    '_isPolyfill_': true,
+    isHTML: trustedTypes.isHTML,
```

Comment:
These properties have to be quoted, otherwise the compiler will rename them, and e.g. `window.trustedTypes.isHTML` will not exist.

Reply:
Oh, thanks for noticing. It was caused by autoformat :/ and I didn't notice it.